### DISCUSSION TITLE
how to have an array of foreign keys (one to many relationship)?

### DISCUSSION THREAD

**Discussion by Nisthar** [supabase/supabase#9271] [Questions] [answered]
I have these 2 tables setup:

`room`:
-id

`groups`:
-id
-members (I want to make this an array of foreign keys pointing to users table)

A room can have multiple groups. and a group can have multiple users.

**Comment by steve-chavez** [comment, marked as answer]
Foreign key arrays are not supported in PostgreSQL, you'd need to create a join table.

See this previous discussion: https://github.com/supabase/supabase/discussions/535#discussioncomment-285696
